currency/pair_test.go

```diff
@@ -337,6 +337,16 @@ func TestNewPairDelimiter(t *testing.T) {
 			actual, expected,
 		)
 	}
+
+	pair = NewPairDelimiter("BTC-MOVE-0626", "-")
```

Comment:
:D interesting little side effect here if you do a mixed case (eg fBTC-USDT) it will lower case the symbol